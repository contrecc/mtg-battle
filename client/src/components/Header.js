import React from 'react';

export default () => {
  return (
    <div style={{ marginTop: '40px', marginBottom: '80px' }}>
      <h1
        style={{ fontSize: '50px', textAlign: 'center' }}
        className="text-info"
      >
        MTG BATTLE
      </h1>
      <p className="lead text-center">
        Guess which creature would defeat the other in combat!
      </p>
    </div>
  );
};
