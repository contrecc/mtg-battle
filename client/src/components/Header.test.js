import Header from './Header';

describe('Header', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
