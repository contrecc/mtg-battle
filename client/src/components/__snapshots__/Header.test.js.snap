// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  style={
    Object {
      "marginBottom": "40px",
      "marginTop": "20px",
    }
  }
>
  <h1
    className="text-info"
    style={
      Object {
        "fontSize": "50px",
        "textAlign": "center",
      }
    }
  >
    MTG BATTLE
  </h1>
  <p
    className="lead text-center"
  >
    Guess which 
    <strong>
      Magic: The Gathering
    </strong>
     creature would defeat the other in combat!
  </p>
</div>
`;
